docs/source/configuration.rst

```diff
@@ -197,6 +197,37 @@ Examples
             message: |
               The pull request base branch has reached end-of-life.
 
+
+.. _relative datetime:
+
+Relative datetime
```

Comment:
```suggestion
.. _relative timestamp:

Relative timestamp
```

Datetime is not a word, timestamp is.

We should also fix the previous type, my bad.